### PR TITLE
display task names on nurse dashboard main page

### PR DESCRIPTION
**Pull request by ericcf**
[#112388529]

### CODE REVIEW

File: app/controllers/nurse_dashboards_controller.rb [hunk 1]
```diff
@@ -4,6 +4,10 @@ def show
     @participants = Participant
                     .where(nurse: current_user)
                     .active
+    @tasks = NurseTask
+             .where(nurse: current_user, participant: @participants)
+             .where(status: NurseTask::STATUSES.active)
+             .group_by(&:participant_id)
```
Comment (michaelwehrley) [inline app/controllers/nurse_dashboards_controller.rb]: Thanks for using `group_by`, I wasn't aware of it.

File: app/models/nurse_task.rb [hunk 2]
```diff
@@ -26,6 +26,10 @@ def self.overdue
     where(arel_table[:overdue_at].lteq(Time.zone.now))
   end
 
+  def to_s
```
Comment (michaelwehrley) [inline app/models/nurse_task.rb]: Are you using this?

**Comment by michaelwehrley** [comment]
Improving spec coverage :+1:

File: app/models/nurse_task.rb (hunk 2 above)
Comment (ericcf) [inline app/models/nurse_task.rb]: Yes, in `app/views/nurse_dashboards/show.html.erb` when it renders the array of tasks.

**Pull request merged**